packages/features/ee/organizations/lib/server/orgCreationUtils.ts

```diff
@@ -204,7 +204,8 @@ export async function assertCanCreateOrg({
   });
 
   // Let slug verification be done before it as that gives better error message if the user creating the org is already a part of the org(i.e. the org exist and he is a member)
-  if (orgOwner.profile?.organizationId) {
+  const isPartOfAnotherOrg = orgOwner.teams.some((team) => team.team.isOrganization);
+  if (isPartOfAnotherOrg) {
```

Comment:
Lets improve the message to inform clearly that it is a Platform or Organizastion

'You are part of Platform organization'

'You are part of Organization'

Also, we need to ensure that membership is in accepted state. Plus, let's add comments as to why we aren;t checking for profile.organizationId.